lib/netsuite/records/non_inventory_sale_item.rb

```diff
@@ -28,12 +28,13 @@ class NonInventorySaleItem
       record_refs :billing_schedule, :cost_category, :custom_form, :deferred_revenue_account, :department, :income_account,
         :issue_product, :item_options_list, :klass, :location, :parent, :pricing_group, :purchase_tax_code,
         :quantity_pricing_schedule, :rev_rec_schedule, :sale_unit, :sales_tax_code, :ship_package, :store_display_image,
-        :store_display_thumbnail, :store_item_template, :subsidiary_list, :tax_schedule, :units_type
+        :store_display_thumbnail, :store_item_template, :tax_schedule, :units_type
 
-      field :pricing_matrix, PricingMatrix
```

Comment:
This was included twice. I dropped the first one so they'd be sorted alphabetically.